ATCBot/SteamConfig.cs

```diff
@@ -0,0 +1,49 @@
+using System.IO;
+using Newtonsoft.Json;
+
+namespace ATCBot
+{
+    public class SteamConfig
```

Comment:
Would it be more efficient to inherit from `Config`?

Reply:
The reason why it is its own class is that I thought it would be better to have sensitive all in one other document instead of the same general config.

Reply:
Fine with me.